app/containers/HomePage/index.js

```diff
@@ -33,68 +33,72 @@ import { makeSelectUsername } from './selectors';
 import reducer from './reducer';
 import saga from './saga';
 
-/* eslint-disable react/prefer-stateless-function */
-export class HomePage extends React.PureComponent {
-  /**
-   * when initial state username is not null, submit the form to load repos
-   */
-  componentDidMount() {
-    if (this.props.username && this.props.username.trim().length > 0) {
-      this.props.onSubmitForm();
-    }
-  }
+const key = 'home';
 
-  render() {
-    const { loading, error, repos } = this.props;
-    const reposListProps = {
-      loading,
-      error,
-      repos,
-    };
+function HomePage({
```

Comment:
Let's add `export` here so the component can be tested without the HOC, if needed. Also, this _should_ get the tests to pass. 😄

Reply:
Good catch. Updating now.

Note: It doesn't fix ALL the tests (notably the ones which check features that rely on useEffect) but I'm gonna wait til we merge #2592 to take care of that.

Reply:
Agreed. Nice job! 👍